### DISCUSSION TITLE
Timeout Actions awaiting review

### DISCUSSION THREAD

**Discussion by dominiquedejaeger-liantis** [community/community#33472] [Actions]
Is it possible to adjust the timeout period of 30 days for Gtihub actions who are awaiting a review / appoval

In the documentation on https://docs.github.com/en/actions/managing-workflow-runs/reviewing-deployments it only states that the actions will timeout.

**Comment by samuel-ma** [comment]
@dominiquedejaeger-liantis 
Unfortunately no....
Once you've selected Github actions, your timeout period is set and u have 30days to await a review / approval

**Comment by benjiwolff** [comment]
We are also missing an option to set a default timeout period.
